sssom/__init__.py

```diff
@@ -8,3 +11,6 @@
     group_mappings,
     parse,
 )
+
+SCHEMA_PARENT_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
```

Comment:
if this is referring to a file outside of the directory structure of the package, it simply won't work for anyone who pip installs it

Reply:
the other reason why it's nice to make constants is because it makes it easier to find bugs like this ^

Reply:
BUt if it is in the same directory as that __init__.py iys fine?